lib/packages/shared-utils/legacy-event-type.ts

```diff
@@ -0,0 +1,56 @@
+import { Action } from "shared-types";
+
+// Resolve the action type based on the GSI1pk
+export const getLegacyEventType = (GSI1pk: string) => {
+  if (!GSI1pk) {
+    return undefined;
+  }
```

Comment:
Is this check for `GSI1pk` being an empty string? Better be more explict: `GSI1pk === 
''`